docs/standard/base-types/custom-numeric-format-strings.md

```diff
@@ -191,23 +195,55 @@ You can create a custom numeric format string, which consists of one or more cus
  The following example uses the ";" format specifier to format positive, negative, and zero numbers differently.  
   
  [!code-cpp[Formatting.Numeric.Custom#8](../../../samples/snippets/cpp/VS_Snippets_CLR/formatting.numeric.custom/cpp/custom.cpp#8)]
- [!code-csharp[Formatting.Numeric.Custom#8](../../../samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/custom.cs#8)]
+ [!code-csharp-interactive[Formatting.Numeric.Custom#8](../../../samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/custom.cs#8)]
  [!code-vb[Formatting.Numeric.Custom#8](../../../samples/snippets/visualbasic/VS_Snippets_CLR/formatting.numeric.custom/vb/Custom.vb#8)]  
   
  [Back to table](#table)  
-  
+
+## Character literals  
+ 
+The following characters in a custom numeric format string are reserved and are always interpreted as formatting characters:  
+
+- 0
+- \#
+- %
+- ‰
+- '
+- \
+- .
+- ,
+- E or e, depending on its position in the format string.
+
+All other characters are always interpreted as character literals and, in a formatting operation, are included in the result string unchanged.  In a parsing operation, they must match the characters in the input string exactly; the comparison is case-sensitive.  
+  
+The following example illustrates one common use of a literal character units (in this case, thousands):
+  
+ [!code-csharp[literal characters](~/samples/snippets/csharp/VS_Snippets_CLR/formatting.numeric.custom/cs/literals2.cs)]
+ [!code-vb[literal characters](~/samples/snippets/visualbasic/VS_Snippets_CLR/formatting.numeric.custom/vb/literals2.vb)]  
+  
+ There are two ways to indicate that characters are to be interpreted as literal characters and not as reserve characters, so that they can be included in a result string or successfully parsed in an input string:  
+  
+- By escaping each reserved character. For more information, see [The "\\" escape character]](#SpecifierEscape).
```

Comment:
The same as above: *reserved* -> *formatting*

Also, replace *each* with *a*, since now it might be read that one must escape either all or zero formatting characters.

Also, double closing square bracket should be replaced with one.

Reply:
Sorry about that, @pkulikov. I accidentally selected the wrong emoji.